### DISCUSSION TITLE
Tag push jobs are not being triggered

### DISCUSSION THREAD

**Discussion by denzp** [community/community#25802] [Actions]
<p>I’ve encountered the problem in the repo: <a href="https://github.com/denzp/cargo-wharf" rel="nofollow noopener">https://github.com/denzp/cargo-wharf</a></p>
<p>There are several workflows defined:</p>
<pre><code>name: Publish Docker Hub description
on:
  push:
    tags:
      - cargo-wharf-frontend-latest
      - cargo-container-tools-latest

name: Publish the cargo-container-tools
on:
  push:
    tags:
      - cargo-container-tools-*

name: Publish the cargo-wharf-frontend
on:
  push:
    tags:
      - cargo-wharf-frontend-*

name: Continuous integration
on:
  pull_request:
    branches:
      - master

  push:
    branches:
      - master
</code></pre>
<p>It works fine with pull requests and commits into master, but I can’t manage to trigger <em>tag</em> jobs.</p>
<p>I’ve tried both lightweight and annotated <a href="https://github.com/denzp/cargo-wharf/tags" rel="nofollow noopener">tags</a> already, no luck with these tags:</p>
<pre><code>cargo-wharf-frontend-v0.1.0-alpha.0
cargo-wharf-frontend-latest
cargo-container-tools-v0.2.0-alpha.0
cargo-container-tools-latest
</code></pre>

**Comment by BrightRan** [comment, marked as answer]
<p>Hi denzp,</p>
<p>I forked your repository and tested the tags filter on the fork repository:</p>
<p>I checkout the branch consistent-caching, and made some changes, then I executed the following git commands to add to tags and push the tags the remote.</p>
<pre><code>git commit -a -m "Update README.md - 19110803"
git tag cargo-container-tools-ver03
git tag cargo-wharf-frontend-ver03
git push –tags
</code></pre>
<p> Then these two tags triggered two workflows as expectation.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/05859f969c8c5b8c1abdf1367a6491f136f6db00.png" data-download-href="/uploads/short-url/MQFNqARymQN32Dz5mpa8S5uXPq.png?dl=1" title="example.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/46307996/181093547-ab9b29de-7250-47dc-a957-84d5870f1d74.png" alt="example.PNG" data-base62-sha1="MQFNqARymQN32Dz5mpa8S5uXPq" width="690" height="171" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/05859f969c8c5b8c1abdf1367a6491f136f6db00_2_690x171.png, https://aws1.discourse-cdn.com/github/optimized/2X/0/05859f969c8c5b8c1abdf1367a6491f136f6db00_2_1035x256.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/0/05859f969c8c5b8c1abdf1367a6491f136f6db00_2_1380x342.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/05859f969c8c5b8c1abdf1367a6491f136f6db00_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">example.PNG</span><span class="informations">1709×426 45.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>This also can work on master branch, but not on experiment-dockerfile branch, because the on experiment-dockerfile branch there are no the YAML files for workflows.</p>
<p>What branch did you checkout to push the tags? Please make sure the YAML files for workflows are existing on the branch.</p>

**Comment by denzp** [comment]
<p>Thank you <a class="mention" href="/brightran">@brightran</a> for trying this out! It worked for me when I deleted the tags from remote and pushed them again today.</p>
